src/o365/spo/commands/listitem/listitem-batch-add.ts

```diff
@@ -0,0 +1,470 @@
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import {
+  CommandOption,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+import Utils from '../../../../Utils';
+//import { ListItemInstance } from './ListItemInstance';
+import { FolderExtensions } from '../../FolderExtensions';
+import * as path from 'path';
+import { Transform } from 'stream';
+const vorpal: Vorpal = require('../../../../vorpal-init');
+const csv = require('@fast-csv/parse');
```

Comment:
Hi, I get typescript errors out of fast-csv if i import it rather than requiring it:

node_modules/@fast-csv/parse/build/src/CsvParserStream.d.ts:18:17 - error TS7033: Property 'hasHitRowLimit' implicitly has type 'any', because its get accessor lacks a return type annotation.

18     private get hasHitRowLimit();
                   ~~~~~~~~~~~~~~

node_modules/@fast-csv/parse/build/src/CsvParserStream.d.ts:19:17 - error TS7033: Property 'shouldEmitRows' implicitly has type 'any', because its get accessor lacks a return type annotation.

19     private get shouldEmitRows();
                   ~~~~~~~~~~~~~~

node_modules/@fast-csv/parse/build/src/CsvParserStream.d.ts:20:17 - error TS7033: Property 'shouldSkipLine' implicitly has type 'any', because its get accessor lacks a return type annotation.

20     private get shouldSkipLine();
                   ~~~~~~~~~~~~~~

Reply:
That's odd, I haven't noticed any of these errors. Do you happen to have typescript installed globally? If so, is it a different version than the one we have in the project? Which version of Node.js are you using?

Reply:
I don't think i do:
λ npm list -g --depth 0                                                              
C:\Users\trwg1\AppData\Roaming\npm                                                   
+-- @microsoft/generator-sharepoint@1.10.0                                           
+-- @pnp/office365-cli@2.7.0 -> C:\Users\trwg1\GitHub\office365-cli-NEW              
+-- azure-functions-core-tools@3.0.2106                                              
+-- create-react-app@3.0.0                                                           
+-- gulp@4.0.2                                                                       
+-- ngrok@3.2.7                                                                      
`-- yo@3.1.1

Reply:
And 
λ node -v
v10.19.0